src/components/OneArticle.jsx

```diff
@@ -100,6 +99,19 @@ function OneArticle() {
       });
   }
 
+  function deleteComment(comment_id) {
+    axios
+      .delete(
+        `https://backend-service-project-2.onrender.com/api/comments/${comment_id}`
+      )
+      .then((response) => {
+        setArticleComments((articleComments) =>
+          articleComments.filter((comment) => comment.comment_id !== comment_id)
+        );
+        alert("Comment successfully deleted");

```

Comment:
I would highly recommend a different way of conveying this message. Alerts are not a great user experience and quite distracting on mobile.